### ISSUE TITLE
Should GridTools::extract_used_vertices return also artificial vertices?

### ISSUE THREAD

**Issue by luca-heltai** [dealii/dealii#7577]
Related to #7576 and #7555.

The function `GridTools::extract_used_vertices` should really return *also* the vertices of the artificial cells, otherwise there is a misnomer somewhere. `Triangulation::get_used_vertices` returns *all* used vertices, regardless of ownership.

The issue with `GridTools::extract_used_vertices` is that it does so respecting the mapping, and a mapping may not work on artificial cells (for example, `MappingQEulerian`).

**Comment by bangerth** [comment]
We can of course rename the function to `extract_locally_relevant_vertices()` or some such. Then the function would at least be appropriately named. The question is what one wants to do with it -- what is the use case you're interested in?

**Comment by kronbichler** [comment]
@luca-heltai I guess we postpone this one to the 9.2 release milestone?

**Comment by luca-heltai** [comment]
Yes. There are way too many problems with all the `find_active_cell_around_points` stuff to hope finishing them in time. 9.2 it is.

**Comment by masterleinad** [comment]
I guess this isn't high-priority for the release. Postponing.
